### PR TITLE
README and Gallery cosmetic changes + Renaming some examples

### PR DESCRIPTION
**Pull request by guillaumebrg**
Change list: 
- Renaming: `gesture_recognition` >> `action_recognition`. This demo supports non-gesture classes, e.g. nodding, shaking head, so I guess this is a better name.
- Renaming & Unifying: `hand gesture event detection` / `gesture_detection` >> `gesture_control`
- Recorded new GIFs because the "Action Recognition" and "Gesture Control" ones were too similar (a lot of swipes in both cases). GIFs were also re-ordered (looked better with the fitness GIF in the middle)
- Added a mosaic GIF to the "Gallery" section of the README
- Added a preview  of the Virtual Race App to the gallery

### CODE REVIEW

**Review by corneliusboehm — COMMENTED** [review]
The renaming makes sense to me :+1: And thanks for adding that cool mosaic!

You seem to have uploaded the weights for the `gesture_control` demo, which were just added to the SDK download page. Was that accidental? If we remove them here again, we need to make sure to at least rename the folder in the downloadable zip file.

File: examples/run_action_recognition.py [hunk 1]
```diff
@@ -65,7 +65,7 @@
     # Create a logistic regression classifier
     gesture_classifier = LogisticRegression(num_in=backbone_network.feature_dim,
                                             num_out=30)
-    gesture_classifier.load_state_dict(weights['gesture_recognition'])
+    gesture_classifier.load_state_dict(weights['action_recognition'])
```
Comment (sunny-panchal) [inline examples/run_action_recognition.py:68]: Should these variables also be renamed to `action_classifier`?

**Review by sunny-panchal — COMMENTED** [review]
As we discussed, it would be nice to re-record the gesture control video. 

Might make sense to rearrange the demo instructions to match the gif order and to add a link to the gallery in the table of contents. I'll add both of those now.

File: examples/run_action_recognition.py (hunk 1 above)
Comment (guillaumebrg) [reply examples/run_action_recognition.py:68]: Yep, good point

File: README.md [hunk 2]
```diff
@@ -147,20 +143,7 @@ Usage:
...
-policy makes it easier to quickly trigger multiple hand gestures in a row. 
-
-Usage:
-```shell
-PYTHONPATH=./ python examples/run_gesture_control.py
-```
-
-
-#### Demo 3: Fitness Activity Tracking
+#### Demo 2: Fitness Activity Tracking
```
Comment (guillaumebrg) [inline README.md:146]: I wanted to leave the fitness tracking section next to the calorie estimation and repetition counting ones? Let me know what you think

**Comment by guillaumebrg** [comment]
> You seem to have uploaded the weights for the gesture_control demo, which were just added to the SDK download page. Was that accidental?

That's accidental... Will remove and talk with Ingo to rename the folder

**Comment by guillaumebrg** [comment]
Ingo already made the changes on the website. Let's merge and address the remaining points in a follow up PR

**Pull request merged**